CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Change Log
 
 ## Master/Unreleased
-
+Add TypeScript type definitions [#146](https://github.com/emlynmac/html-to-react/pull/146) ([xnevz](https://github.com/xnevz), [aknuds1](https://github.com/aknuds1),[emlynmac](https://github.com/emlynmac))
```

Comment:
Please add a blank line before and after the entry, and change github.com/emlynmac into github.com/aknuds1.

Reply:
Feel free to update any stylistic changes as needed.

Reply:
I'm sorry, but that's not my job.